src/geometry/ClosedPolyline.cpp

```diff
@@ -54,4 +55,24 @@ OpenPolyline ClosedPolyline::toPseudoOpenPolyline() const
     return open_polyline;
 }
 
+void ClosedPolyline::rotateToStartPoint(const size_t start_index)
```

Comment:
Can we rename this to `indexRotateToStartPoint`. When I read "rotate" in the context of polygons I think of something different.